operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -179,6 +179,7 @@ mimir:
       {{- if .Values.query_scheduler.enabled }}
       scheduler_address: {{ template "mimir.fullname" . }}-query-scheduler-headless.{{ .Release.Namespace }}.svc:{{ include "mimir.serverGrpcListenPort" . }}
       {{- end }}
+      parallelize_shardable_queries: true
```

Comment:
```suggestion
      parallelize_shardable_queries: true
      {{- if .Values.query_scheduler.enabled }}
      scheduler_address: {{ template "mimir.fullname" . }}-query-scheduler-headless.{{ .Release.Namespace }}.svc:{{ include "mimir.serverGrpcListenPort" . }}
      {{- end }}
```

Reply:
done in https://github.com/grafana/mimir/pull/2655/commits/e5b2b1a7dabf47d8190d052726e5bdef9be763ee